pythonforandroid/bootstraps/common/build/src/main/java/org/kivy/android/PythonService.java

```diff
@@ -18,6 +18,12 @@
 
 import org.renpy.android.Hardware;
 
+//new imports for channel definition
```

Comment:
they are new now, but won't be anymore in some time, we can keep the part about them being needed for channel definition.

Reply:
god point